doc/gpu.md

```diff
@@ -22,16 +29,19 @@ interleaved). Anything else will return an error.
 
 * **CUDA Availability** If running the Livepeer binary, the CUDA shared libraries are expected to be installed in `/usr/local/cuda`. If the CUDA location differs on your machine, run the node with `LD_LIBRARY_PATH=</path/to/cuda>` environment variable.
 
-So far, Livepeer has been tested  to work with the following driver versions:
+So far, Livepeer has been tested to work with the following driver versions:
 
 CUDA | Nvidia
 --|--
 10.0.130 |
 10.1 | 418.39 , 430.50
-10.2.89 | 440.33.01
```

Comment:
What's the rationale for removing references to these CUDA versions?

Reply:
Combined this particular one with 440.118.02 (which was also tested with CUDA version 10.2.xx in BER - not sure what xx was). Most users can anyway not control the minor version's of CUDA/Nvidia libs that are available for them to install.

Removed `11.0 | 450.36.06` because 450 occasionally lead to stuck sessions.